rocketpool/api/network/rpl-price.go

```diff
@@ -49,30 +46,23 @@ func getRplPrice(c *cli.Context) (*api.RplPriceResponse, error) {
 		rplPrice, err = network.GetRPLPrice(rp, nil)
 		return err
 	})
-	wg.Go(func() error {
-		var err error
-		minPerMinipoolStake, err = protocol.GetMinimumPerMinipoolStakeRaw(rp, nil)
-		return err
-	})
 
 	// Wait for data
 	if err := wg.Wait(); err != nil {
 		return nil, err
 	}
 
-	// Min for LEB8s
-	minPer8EthMinipoolRplStake := big.NewInt(0)
-	minPer8EthMinipoolRplStake.Mul(_24Eth, minPerMinipoolStake) // Min is 10% of borrowed (24 ETH)
-	minPer8EthMinipoolRplStake.Div(minPer8EthMinipoolRplStake, rplPrice)
-	minPer8EthMinipoolRplStake.Add(minPer8EthMinipoolRplStake, big.NewInt(1))
-	response.MinPer8EthMinipoolRplStake = minPer8EthMinipoolRplStake
+	// RPL stake amounts for 5,10,15% borrowed ETH per LEB8
+	fivePercentBorrowedPerMinipool := new(big.Int)
+	fivePercentBorrowedPerMinipool.SetString("50000000000000000", 10)
 
-	// Min for 16s
-	minPer16EthMinipoolRplStake := big.NewInt(0)
-	minPer16EthMinipoolRplStake.Mul(_16Eth, minPerMinipoolStake) // Min is 10% of borrowed (16 ETH)
-	minPer16EthMinipoolRplStake.Div(minPer16EthMinipoolRplStake, rplPrice)
-	minPer16EthMinipoolRplStake.Add(minPer16EthMinipoolRplStake, big.NewInt(1))
-	response.MinPer16EthMinipoolRplStake = minPer16EthMinipoolRplStake
+	fivePercentBorrowedRplStake := big.NewInt(0)
+	fivePercentBorrowedRplStake.Mul(_24Eth, fivePercentBorrowedPerMinipool)
+	fivePercentBorrowedRplStake.Div(fivePercentBorrowedRplStake, rplPrice)
+	fivePercentBorrowedRplStake.Add(fivePercentBorrowedRplStake, big.NewInt(1))
+	response.FivePercentBorrowedRplStake = fivePercentBorrowedRplStake
+	response.TenPercentBorrowedRplStake = new(big.Int).Mul(fivePercentBorrowedRplStake, big.NewInt(2))
```

Comment:
Just return the price and borrowed amount, and make the CLI calculate the percent of borrowed. This is an example of what I mean when I say the API over-fits the cli.